mne/io/brainvision/tests/data/test_highpass_hz.vhdr

```diff
@@ -0,0 +1,103 @@
+Brain Vision Data Exchange Header File Version 1.0
+; Data created by the Vision Recorder
```

Comment:
That's what I have been rapidly discovering -- `difflib` isn't as useful here as you would hope.